src/Program.ts

```diff
@@ -283,36 +283,38 @@ export class Program {
      * Get the type definitions for the specified file.
      * The results are cached for future performance boosts.
      */
-    private async getTypeDefinitionsForFile(pathAbsolute: string) {
+    private async getTypedefsForFile(pathAbsolute: string) {
```

Comment:
What if the file is a `d.bs` file already?

Reply:
That's not currently possible. In `Program.addOrReplaceFile`, there's an [if branch](https://github.com/rokucommunity/brighterscript/blob/4be26ae9e5691446d4bb81338e4490260338ff44/src/Program.ts#L366) that refreshes the `typedefCache` for `d.bs` files. So this function won't be called for d.bs files.

Reply:
This comment is no longer relevant...the typedefs are added as actual files into the program like other files.